lib/routes/dataguidance/index.ts

```diff
@@ -10,7 +10,7 @@ export const route: Route = {
     path: '/news',
     radar: [
         {
-            source: ['www.dataguidance.com/info'],
+            source: ['www.dataguidance.com/info?article_type=news_post'],
```

Comment:
> `source` cannot match URL search parameters

_Originally posted by TonyRL in https://github.com/DIYgod/RSSHub/pull/17695#discussion_r1855419962_